server/api/impl/upstream/upawslambda/upstream.go

```diff
@@ -100,3 +118,20 @@ func callToInvocationPayload(call *apps.Call) ([]byte, error) {
 
 	return payload, nil
 }
+
+func match(callPath string, app *apps.App) (string, error) {
+	matchedName := ""
+	matchedPath := ""
+	for _, f := range app.Functions {
+		if strings.HasPrefix(callPath, f.Path) {
+			if len(f.Path) > len(matchedPath) {
```

Comment:
`matchedPath` assignment is missing

Reply:
Oops, had a favorable order in the test. Fixed.